### PR TITLE
refactor: remove experimental references from documentation

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Supports the discontinuation of the Experimental Version

### Location
- Multiple Locations based on references

### TODO

Working top level directories to tackle:

- [x] fbw-a32nx - WIP Val
- [x] dev-corner
- [x] pilots-corner
- [x] release-notes
- [x] simbridge
- [x] Verify mention of FWS initialization is correct
- [x] Fix Installation.md changelog link to go to /release-notes directory instead

Check when complete.

Discord username (if different from GitHub): valastiri

### CODE REVIEW

**Comment by Valastiri** [comment]
Updated the config to kill the redirects

File: docs/fbw-a32nx/faq.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
     Just make sure to search for existing issues first before creating a new one.
 
 ??? info "Q: Why is my version not the same as what I see others using?"
-    We have three versions: Stable, Development, and Experimental.
+    We have two versions: Stable and Development
```
Comment (Lauvaan) [inline docs/fbw-a32nx/faq.md]: Missing dot(not actually sure if it needs one, but the previous text had one)

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The SRS guidance law also includes:
 
 ## CLB (Climb)
 
-!!! warning "Managed CLB or DES (VNAV) is currently only available in the Development & Experimental versions."
+!!! warning "Managed CLB or DES (VNAV) is currently only available in the Development version."
```
Comment (Lauvaan) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]: "version" should have a capital "V", to be in line with other instances in the docs

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ When ALT is engaged, the FMA displays ALT in green (FCU altitude hold), ALT CST
 
 ## DES (Descent)
 
-!!! warning "Managed CLB or DES (VNAV) is currently only available in the Development & Experimental versions."
+!!! warning "Managed CLB or DES (VNAV) is currently only available in the Development version."
```
Comment (Lauvaan) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]: "version" should have a capital "V", to be in line with other instances in the docs

File: docs/pilots-corner/beginner-guide/descent.md [hunk 4]
```diff
@@ -250,7 +250,7 @@ A few minutes before we reach our calculated descent point (TOD) we request clea
 !!! info "TOD marker A320"
     The A320 has a downward pointing arrow at the TOD to support the pilot with the decision of when to descend. Ultimately, it is still the pilot's responsibility to calculate and validate the TOD.
 
-    The TOD (top of descent) marker on the `ND` is available in the Development and Experimental version. 
+    The TOD (top of descent) marker on the `ND` is available in the Development version.
```
Comment (Lauvaan) [inline docs/pilots-corner/beginner-guide/descent.md]: "version" should have a capital "V", to be in line with other instances in the docs

File: docs/pilots-corner/beginner-guide/descent.md [hunk 5]
```diff
@@ -278,7 +278,7 @@ Also, ATC will often still expect us to respect the STAR's constraints, although
     The scenario that we are cleared to a lower altitude or flight level with altitude constraints above the clearance is an ideal scenario for the called so "VNAV" autopilot mode, which would be activated by using "`Managed Altitude Mode`" (pushing the `ALT selector`). The autopilot will automatically level off at the constraint and continue descending when the constraint is no longer valid.
 
     !!! warning ""
-        VNAV is currently only supported in our Development and Experimental versions of the A32NX.
+        VNAV is currently only supported in our Development version of the A32NX.
```
Comment (Lauvaan) [inline docs/pilots-corner/beginner-guide/descent.md]: "version" should have a capital "V", to be in line with other instances in the docs

**Review by Lauvaan — CHANGES_REQUESTED** [review]
Some minor things. Not 100% sure about the capital "V"'s, as in those instances, that were changed they were not capital previously, but in some other places they were.

**Comment by Sleinmaster** [comment]
> Some minor things. Not 100% sure about the capital "V"'s, as in those instances, that were changed they were not capital previously, but in some other places they were.

Actually, the first letter of development and stable should be lowercase.

**Comment by Lauvaan** [comment]
> Actually, the first letter of development and stable should be lowercase.

So the word "development" should be lowercase? Or the "version" should be lowercase?

**Comment by frankkopp** [comment]
> > Some minor things. Not 100% sure about the capital "V"'s, as in those instances, that were changed they were not capital previously, but in some other places they were.
> 
> 
> 
> Actually, the first letter of development and stable should be lowercase.

Stable, Development and Experimental are edition names, and therefore capitalized.

**Comment by Lauvaan** [comment]
Well, no need to worry about the capitalization in this pr imo. Looking a bit more carefully through the docs, it seems that there are many places where neither the edition name nor "version(s)" have been capitalized, and then many occasions where both of them or only on of them is capitalized. This could be a thing for another pr sometime somewhen.

**Comment by frankkopp** [comment]
> Well, no need to worry about the capitalization in this pr imo. Looking a bit more carefully through the docs, it seems that there are many places where neither the edition name nor "version(s)" have been capitalized, and then many occasions where both of them or only on of them is capitalized. This could be a thing for another pr sometime somewhen.

That's fine. But they should be capitalized. If not it is an error. I always tried to make sure to be consistent with it.

**Comment by Sleinmaster** [comment]
> > > Some minor things. Not 100% sure about the capital "V"'s, as in those instances, that were changed they were not capital previously, but in some other places they were.
> > 
> > 
> > Actually, the first letter of development and stable should be lowercase.
> 
> Stable, Development and Experimental are edition names, and therefore capitalized.

Yeah, looking at it this way that's probably right, although I didn't really find any rules about it. 
Versions with the s at the end should definitely be lowercase, probably without the s as well.

So I would say the ones commented on by lauvaan are correct, if there are other capitalizations somewhere they should at some point be corrected.

**Comment by Valastiri** [comment]
Ah yea sorry was super rushed -- as Mav said as a team we decided edition names are captilalized but in terms of the word version I think I'll take lauvan's recommendation for now and we can look at it in another PR

**Comment by Valastiri** [comment]
returned exp.md with a simple statement to support the marketing website's "expOnHold" copy that links back to our documentation.

**Pull request merged**